src/map/clif.c

```diff
@@ -13161,6 +13181,10 @@ bool clif_sub_guild_invite(int fd, struct map_session_data *sd, struct map_sessi
 		return false;
 	}
 
+	// Players in a clan can't join a guild
+	if (t_sd->clan)
```

Comment:
This should probably send a message just like the others above, "You can't join in a clan if you're in a guild."